MAUI/DataGrid/selection.md

```diff
@@ -0,0 +1,547 @@
+</td>
+</tr>
+<tr>
+<td>
+<kbd>Enter</kbd>
+</td>
+<td>
+If the active current cell is in edit mode, the changes will committed and moves the current cell to below the active current cell. If the active current cell is in last row, commits changes only and retains in the same cell.
+</td>
+</tr>
+<tr>
+<td>
+<kbd>Ctrl</kbd> + <kbd>A</kbd>
+</td>
+<td>
+All rows or cells will be selected.
+</td>
+</tr>
+</table>
+
+## Move Current Cell
+The [CurrentCell] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CurrentCell) can be moved to a particular [RowColumnIndex] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.GridCommon.ScrollAxis.RowColumnIndex.html) by using the [SfDataGrid.MoveCurrentCellTo()] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_MoveCurrentCellTo_Syncfusion_Maui_GridCommon_ScrollAxis_RowColumnIndex_System_Boolean_) method. This method is not applicable when the `SfDataGrid.SelectionMode` is None or `NavigationMode` is Row.
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        dataGrid.MoveCurrentCellTo(new RowColumnIndex(3, 1), false);
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Move current cell in .NET MAUI DataGrid](Images/selection/maui-datagrid-movecurrentcell.png)
+
+## Clear selection
+`DataGrid` allows you to clear the selection applied in the grid rows and columns either by setting the `SfDataGrid.SelectionMode` to None or by calling the [SfDataGrid.ClearSelection()] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_ClearSelection) method.
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        //Clear selection using selection mode
+        dataGrid.SelectionMode = Syncfusion.Maui.DataGrid.DataGridSelectionMode.None;
+
+        //Clear selection using clear selection method.
+        dataGrid.ClearSelection();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Events in selection
+The DataGrid provides the following events for selection:
+
+[SelectionChanging] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SelectionChanging): This event is raised while selecting a row at the execution time before the row is selected. So it allows canceling the selection action by setting the [Cancel] (https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?view=net-6.0#system-componentmodel-canceleventargs-cancel) property of [DataGridSelectionChangingEventArgs] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSelectionChangingEventArgs.html).
```

Comment:
- [SelectionChanging] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SelectionChanging): This event is raised while selecting a row at the execution time before the row is selected. So it allows canceling the selection action by setting the [Cancel] (https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?view=net-6.0#system-componentmodel-canceleventargs-cancel) property of [DataGridSelectionChangingEventArgs] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSelectionChangingEventArgs.html).